src/sentry/tasks/auto_review_inbox.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import absolute_import
+
+from datetime import timedelta
+from django.utils import timezone
+
+from sentry.models import GroupInbox, Organization, Project, OrganizationStatus
+from sentry.tasks.base import instrumented_task
+
+
+@instrumented_task(
+    name="sentry.tasks.schedule_auto_review_inbox", time_limit=65, soft_time_limit=60
+)
+def schedule_auto_review_inbox():
+    for organization in Organization.objects.filter(status=OrganizationStatus.ACTIVE).iterator():
+        for project in Project.objects.filter(organization=organization).values("id"):
+            auto_review_inbox.delay(project["id"])
+
+
+@instrumented_task(name="sentry.tasks.auto_review_inbox", time_limit=65, soft_time_limit=60)
+def auto_review_inbox(project_id, chunk_size=1000):
+    cutoff = timezone.now() - timedelta(days=7)
+
+    queryset = list(
+        GroupInbox.objects.filter(date_added__lte=cutoff, project=project_id).values_list(
```

Comment:
We still don't have any useful index here, so we're going to want to revisit this with indices before we even attempt this.

Reply:
I've done tests on truncating datetime columns in the past and didn't get any significant difference since it's still the same size btree, just shaped differently. It doesn't end up resulting in any optimization. It still has the same amount of leaf nodes, etc.